engine/release-notes/23.0.md

```diff
@@ -42,6 +42,9 @@ For a full list of pull requests and changes in this release, refer to the relev
   - There are differences in how BuildKit and the legacy builder handle multi-stage builds. For more information, see
     [Multi-stage builds](../../build/building/multi-stage.md#differences-between-legacy-builder-and-buildkit).
 - Add support for pulling `zstd` compressed layers. [moby/moby#41759](https://github.com/moby/moby/pull/41759), [moby/moby#42862](https://github.com/moby/moby/pull/42862)
+- Change the `--containerd` option to signify an externally managed containerd at a socket path. [moby/moby#42373](https://github.com/moby/moby/pull/42373)
```

Comment:
This PR (https://github.com/moby/moby/pull/42373) only move the systemd unit back to the moby repo; docker 20.10 already uses that configuration

Reply:
The main difference is that --containerd was added to the systemd unit

Reply:
Slightly longer explanation; https://github.com/moby/moby/issues/44912#issuecomment-1415374072

> I don't think this changed in the 23.0 release; the packaged versions of `docker-ce` already used this configuration since docker 18.09;
> 
> * [[18.09 backport] systemd: set --containerd socket patch to prevent race-condition docker/docker-ce-packaging#298](https://github.com/docker/docker-ce-packaging/pull/298)
> 
> The systemd unit in this repository were no longer used at that time (but were left in the repository in case someone would find them useful). [009639e](https://github.com/moby/moby/commit/009639e0f46d270309821998836fb664046b3e07) was part of this PR:
> 
> * [Upstream systemd unit files from docker-ce-packaging repo #42373](https://github.com/moby/moby/pull/42373)
> 
> Which brought back the systemd service into this repository, but effectively no change in the packages (other than the packaging scripts now consuming the systemd unit(s) from this repository instead of from the packaging repository).
> 
> Note that _without_ the `--containerd` option, the daemon would already _prefer_ a running containerd service (if it finds a containerd socket in the default location), but this could sometimes cause a race condition if the containerd _service_ started, but the socket not yet created, which could lead to _two_ instances of containerd being started. The `--containerd` flag was added to prevent this race condition.

Reply:
OK, I see, the systemd unit in this repo was outdated. We were using the one from contrib/ in Buildroot.